commands/Moderation/mute.js

```diff
@@ -0,0 +1,59 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRoleId, reportChannelId, titleURL, muteRoleName } = msg.guild.settings;
+  try {
+    const value = msg.mentions.members.first().roles.get(muteRoleId);
```

Comment:
Why are you parsing this manually? I shall point you to the docs. https://komada.js.org/command-arguments.html